content/en/docs/setup/install/istioctl/index.md

```diff
@@ -41,10 +41,10 @@ Kubernetes configuration. The `default` profile is a good starting point
 for establishing a production environment, unlike the larger `demo` profile that
 is intended for evaluating a broad set of Istio features.
 
-If you want to secure Istio control plane service endpoints on top of the `default` profile, you can set the security related configuration parameters:
+If you want to enable mesh wide mutual TLS on top of the `default` profile, you can set the security related configuration parameters:
 
 {{< text bash >}}
-$ istioctl manifest apply --set values.global.controlPlaneSecurityEnabled=true
+$ istioctl manifest apply --set values.global.mtls.enabled=true
```

Comment:
I see this was change in #6593 recently from mtls.enabled.

Reply:
Thanks, I tried to update with another example.